NuGet.config

```
@@ -9,7 +9,7 @@
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <!--  Begin: Package sources from dotnet-emsdk -->
-    <add key="darc-pub-dotnet-emsdk-2a1a284" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-2a1a284c/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-emsdk-7fa7119" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-7fa7119c/nuget/v3/index.json" />
```

Reverting the unexpected change in Nuget.config. Unsure why it was brought along, but we don't want it.
Since we're squashing the PR, the dependency flow commit won't show up.

```suggestion
    <add key="darc-pub-dotnet-emsdk-2a1a284" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-2a1a284c/nuget/v3/index.json" />
```